calico_containers/pycalico/util.py

```diff
@@ -10,3 +26,40 @@ def generate_cali_interface_name(prefix, ep_id):
     if len(prefix) > 4:
         raise ValueError('Prefix must be 4 characters or less.')
     return prefix + ep_id[:11]
+
+
+def get_host_ips(version=4, exclude=[]):
+    """
+    Gets all IP addresses assigned to this host.
+
+    This function is fail-safe and will return an empty array instead of
```

Comment:
You'll want to include that this function doesn't return loopback addresses in the docstring.